### PR TITLE
LIC5 to issue #6

### PR DESCRIPTION
**Pull request by yani122**
lic5

### CODE REVIEW

**Review by avatar46 — APPROVED** [review]
LGTM

File: src/main/java/com/group24/decide/LIC.java [hunk 1]
```diff
@@ -119,8 +119,18 @@ public boolean Condition4(){
      * @return Return true if the condition is met, other
      * returns false.
      */
-    public static boolean Condition5(){
-        return true;
+    public  boolean Condition5(){
+        for(int idx=0; idx < this.numberPoints -1; idx++){
+                Datapoints a = this.points[idx];
+                Datapoints b = this.points[idx+1];
+                double diff = Utility.difference(b,a);
```
Comment (persman96) [inline src/main/java/com/group24/decide/LIC.java:126]: Wrong indentation here

File: src/main/java/com/group24/decide/Utility.java [hunk 2]
```diff
@@ -15,6 +15,19 @@ public static double calcEuclideanDistance(Datapoints a, Datapoints b){
...
+    /**
+     * Calculate the difference
+     * @param a First datapoint
+     * @param b Second datapoint
+     * @return difference
+     */
+    public static double difference(Datapoints a, Datapoints b) {
+         
+       return a.x -b.x;
+    }
```
Comment (persman96) [inline src/main/java/com/group24/decide/Utility.java:28]: Not sure if a separate function for such a simple calculation is needed

**Review by persman96 — APPROVED** [review]
Looks good! I added some comments that should be addressed but I think it is good enough to merge now anyway.

**Pull request merged**